File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/PdExtentLookup.cs

namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class PdExtentLookup
    {
        public int PdExtentLookupId { get; set; }
        public decimal Minimum { get; set; }
        public decimal Average { get; set; }
        public decimal Maximum { get; set; }
        public System.DateTime StartDate { get; set; }
        public System.DateTime EndDate { get; set; }
        public bool IsActive { get; set; }
        public bool IsDeleted { get; set; }
        public string CreatedBy { get; set; }
        public System.DateTime CreatedDate { get; set; }
        public string ModifiedBy { get; set; }
        public string ModifiedDate { get; set; }

    }
}

File: Web/RMA-Web-Shell/projects/claimcare/src/app/claim-manager/shared/entities/pd-extent-lookup.ts
export interface PdExtentLookup {
    pdExtentLookupId: number;
    minimum: number;
    average: number;
    maximum: number;
    startDate: Date;
    endDate: Date;
    isActive: boolean;
    isDeleted: boolean;
    createdBy: string;
    createdDate: Date;
    modifiedBy: string;
    modifiedDate: string;
}
